<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeWin</class>
 <widget class="QWidget" name="ChargeWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="AmountInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>enter amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge you're acount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChargeBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CHARGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>AmountInput</zorder>
  <zorder>label</zorder>
  <zorder>ChargeBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>BackBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
